Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Guns/Attachments/stock_attachments.yml

```
@@ -846,6 +846,21 @@
     - RMCAttachmentStock
     - RMCAttachmentHG3712Stock
 
+- type: entity
+  parent: RMCStockAttachmentBase
+  id: RMCAttachmentM3714stock
+  name: M3714 stock
+  description: The non-detachable stock of a M37-14 pump shotgun.
+  components:
+  - type: Sprite
+    state: M3717Stock
```

```suggestion
    state: M3714Stock
```
You forgot to rename state.